index.html

```diff
@@ -6,34 +6,37 @@
   <script src="script.js"> </script>
   <link rel="stylesheet" href="styles.css">
 </head>
-
 <body>
   <center>
     <h1>Programming</h1>
   </center>
   <div class="row">
-    <div class="column1" style="background-color:rgb(221, 220, 231);">
+    <div class="column1" style="background-color:rgb(0, 0, 0);">
```

Comment:
should try to stay away from inline styling in elements if we have a separate styling file. this way it's easier to see what is doing what if everything is in one place. so move the "background-color:rgb(0, 0, 0)" to styles.css